hotsos/defs/scenarios/openstack/octavia/bugs/lp2029857.yaml

```diff
@@ -0,0 +1,37 @@
+checks:
+  has_2029857:
+    input:
+      path: 'var/log/apache2/octavia_error.log'
+    start: '(\[.+\]+) ([\d-]+) ([\d:]+\.[\d]+) .+ Unable to load provider
+driver ovn due to'
+    end: '(\[.+\]+) (octavia.common.exceptions.ProviderNotFound: Provider 
+''ovn'' was not found\.)'
+  amphora_provider_enabled:
+    config:
+      handler: 'hotsos.core.plugins.openstack.OpenstackConfig'
+      path: 'etc/octavia/octavia.conf'
+      assertions:
+        - section: api_settings
+          key: enabled_provider_drivers
+          ops: [[contains, amphora]] 
+  ovn_provider_enabled:
```

Comment:
This is fine as-is but you can combine these two checks as:
```
  amphora_ovn_provider_enabled:
    config:
      handler: 'hotsos.core.plugins.openstack.OpenstackConfig'
      path: 'etc/octavia/octavia.conf'
      assertions:
        - section: api_settings
          key: enabled_provider_drivers
          ops: [[contains, amphora]]
        - section: api_settings
          key: enabled_provider_drivers
          ops: [[contains, ovn]]
```

Ideally it'd look like (this one doesn't work):
```
  amphora_ovn_provider_enabled:
    config:
      handler: 'hotsos.core.plugins.openstack.OpenstackConfig'
      path: 'etc/octavia/octavia.conf'
      assertions:
        - section: api_settings
          key: enabled_provider_drivers
          varops: [[contains, amphora], [contains, ovn]]
```

Maybe @dosaboy  knows how to combine multiple 'contains' into one.

Reply:
done (opt 1)